com.discordapp.Discord.appdata.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="0.0.4" encoding="UTF-8"?>
```

Comment:
Should be reverted. The `version` tag here references the XML specification version, not the application's one.

Reply:
Ups my mistake